index.html

```diff
@@ -376,35 +381,39 @@
           </ol>
         </li>
       </ol>
-      <p class="note">`requestIdleCallback` only schedules a single callback,
-      which will be executed during a single <a>idle period</a>. If the
-      callback cannot complete its work before the given <var>deadline</var>
-      then it should call `requestIdleCallback` again (which may be done from
-      within the callback) to schedule a future callback for the continuation
-      of its task, and exit immediately to return control back to the [event
-      loop].</p>
+      <p class="note"><code><a data-link-for="Window">requestIdleCallback</a>
```

Comment:
fwiw, I think you can pull the data-link-for's up into the `<p data-link..>` (or even higher in the tree) and just `<a>requestIdleCallback</a>`.